Cargo.toml

```diff
@@ -12,11 +12,12 @@ log = "0.4.11"
 rustc-demangle = "0.1.16"
 structopt = "0.3"
 clap = { version = "2.33", default-features = false }
+regex = "1"
 
 [dependencies.haybale]
 #git = "https://github.com/hudson-ayers/haybale"
 #branch = "partitioning"
-path = "/home/hudson/code_repos/haybale" # comment in when testing local haybale development
+path = "/home/paul/haybale" # comment in when testing local haybale development
```

Comment:
Have you been needing to make changes to haybale locally as part of this work, or did you just do this rather than switch to the git dependency?